### ISSUE TITLE
plugin list command error

### ISSUE THREAD

**Issue by navidshaikh** [knative/client#314]
> /kind good-first-issue
> /kind bug

## What version of Knative Client?
latest HEAD

## Expected Behavior
Display clear error about non existent `~/.kn/plugins` dir.

## Actual Behavior
Repetitive error messages

```
kn plugin list
Unable read directory '/home/nshaikh/.kn/plugins' from your plugins path: open /home/nshaikh/.kn/plugins: no such file or directory. Skipping...
warning: unable to find any kn plugins in your plugin path: '[/home/nshaikh/.kn/plugins]'
```

**Comment by navidshaikh** [comment]
@rhuss : Probably a good fit for #313 ?

**Comment by rhuss** [comment]
Good point. I added some fixes already, but might have missed that.

**Comment by maximilien** [comment]
OK if #313 does not I can tackle this. I think #313 is almost ready to merge...

**Comment by rhuss** [comment]
Just verified that is already fixed with #313 , however found another issue in combination with --verbose. Going to fix that now.

**Comment by rhuss** [comment]
Fixed it now in #313 so that if no ~/.kn/plugins exists then

```
$ kn plugin list --verbose

The following plugins are available, using options:
  - plugins dir: '~/.kn/plugins' (does not exist)
  - lookup plugins in path: 'false'
No plugins found in path /Users/roland/.kn/plugins
```

Without `--verbose` you just get an empty output (as there are no plugins installed)

**Comment by navidshaikh** [comment]
Fixed with #313
